### DISCUSSION TITLE
New user: Is editing of fingering remarks possible after conversion with Audiveris ?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [Audiveris/audiveris#724] [Q&A]
I am new to OMR software. Need it for importing simple hobby piano sheets and exporting MusicXML to Dorico Elements for editing.

**I like to achieve the following:**
In my learning sheets the fingering is printed on top of the staff.
I like to get this into Dorico Elements with the option to remove or edit it.
Very simple example:
![Notes_Fingering](https://github.com/Audiveris/audiveris/assets/115958660/075d7e6e-5170-42c5-b8f0-c134d3c156ac)

I don't need the explanatory text above the staffs (the example screenshot shows it but the real practice pieces don't have this additional text - there it is just the staffs (but a bit more complex notes of course).

I clicked on releases in github and am offered:
5.3.1 latest and 5.4.-alpha

Would it be possible to achieve my goal with one of those alternatives?

I read that both, Audiveris and Dorico, support MusicXML 4.0 - so I assume that it the exchange format to use?

Input format would be PDF or png in good quality.

**Comment by cgidesign-de** [comment]
I tried it, but no luck so far.

Is there a way to increase the text recognition?
And I don't understand why beat 3 is marked as error?

![06_Audiveris-5 4-alpha_Result](https://github.com/Audiveris/audiveris/assets/115958660/0fc9931f-15db-43ed-9725-e7ee20b51eed)

**Comment by hbitteur** [comment]
I can't think of any way to improve the OCR of fingering text.
I suppose you have already selected the topic "`Fingering digits`" in the menu `Book | Set book parameters`
See also [this section](https://audiveris.github.io/audiveris/_pages/specific/fingering_plucking/).

Regarding measure 3, we can suspect the whole notes not properly merged into a single chord.
Please see [this handbook section](https://audiveris.github.io/audiveris/_pages/ui_tools/chords/#merge) about chord of wholes.
UPDATE: You should also select the topic `multi-whole head chords` in the book parameters

**Comment by aum7** [comment]
quote:
"And I don't understand why beat 3 is marked as error?"
i had repeating similar notes
therefore i can confirm with quite a certainty : 
bass lines > 1st (whole) note should be either deleted
or deleted and put back a bit closer to other 2 whole notes (which are vertically aligned)
if one adds new whole note, by dragging from right panel,
while still holding the left mouse button
when you are 'on the right spot' (meaning bar will be correct) the dragged note 'connects' to the other 2 notes
